src/renderer/widgets/CreateWallet/ui/MultisigWallet/NameNetworkSelection.tsx

```diff
@@ -78,12 +80,15 @@ export const NameNetworkSelection = ({ onGoBack }: Props) => {
             {t('createMultisigAccount.backButton')}
           </Button>
           <div className="mt-auto flex items-center justify-end">
-            <MultisigCreationFees
-              api={api}
-              asset={chain.value.assets[0]}
-              threshold={threshold.value}
-              transaction={fakeTx}
-            />
+            {nonNullable(chain) ? (
+              <MultisigCreationFees
+                api={api}
+                asset={chain.assets[0]}
```

Comment:
can be null in terms of types (someday i'll turn this check on)

Reply:
You mean `assets[0]`? All chains have at least 1 element is assets
To be super strict we should use `.at()` and always check for null this way

Reply:
let's use `.at(0)`